### ISSUE TITLE
UUID on M2M fields

### ISSUE THREAD

**Issue by lokeshwarobuli** [jazzband/django-simple-history#1400]
Is there any way to have UUID as primary key M2M fields that tracks the histories, right now I tried setting `SIMPLE_HISTORY_HISTORY_ID_USE_UUID ` to True as well as tried https://django-simple-history.readthedocs.io/en/latest/historical_model.html#custom-history-id, but still the historical intermediate M2M table only has integer as primary key. Can someone provide some direction in case if you have already gone through similar situation

Thanks

**Comment by lokeshwarobuli** [comment]
bump

**Comment by epiben** [comment]
Indeed, this would be useful. It's also a pity that Django doesn't support this for the M2M tables unless you explicitly define one yourself. If useful to others, I'm happy to give this a shot.
